### DISCUSSION TITLE
DateField shows timezone in forms

### DISCUSSION THREAD

**Discussion by alexanderchr** [laravel/nova-issues#4626] [Ideas & Feature Requests] [answered]
- Laravel Version: 9.16.0
- Nova Version: 4.12.12
- PHP Version: 8.1.7
- Database Driver & Version: NA
- Operating System and Version: NA
- Browser type and version: NA
- Reproduction Repository: NA

### Description:

The DateField shows the timezone when it's being edited:

<img width="278" alt="Screen Shot 2022-07-24 at 13 15 27" src="https://user-images.githubusercontent.com/2178877/180644466-19f27e70-d988-4a85-8ca8-831ecae9efa3.png">

However, there does not seem to be any handling of timezones. The date is written to (and read from) the database as it appears to the user. Therefore showing the timezone next to the date adds nothing but confusion. It should be removed, or the documentation should be updated to make it clear what the meaning of timezone is for this field.

### Detailed steps to reproduce the issue on a fresh Nova installation:

Create a resource with a date field.

**Comment by crynobone** [comment, marked as answer]
This has been solved in 4.26.0

**Reply by alexanderchr** [reply]
Thank you!
